js/tiles.js

```diff
@@ -907,7 +907,7 @@ const door = {
         },
         createImages: function() {
             this.wallTop = this.createImage("tiles/rooms/door/top.svg");
-            this.wallRight = this.createImage("tiles/rooms/door/right.svg");
+            this.wallRight = this.createImage("tiles/rooms/wall/right.svg");
```

Comment:
If you show a wall, the implementation should reflect this. Please also change canLeaveToTheRight and canEnterFromTheRight to false

However, this does not change that the treasure tile is closed from the left where itself has no wall. 
https://github.com/fossasia/labyrinth/blob/master/js/tiles.js#L561
To fix the issue, we would need to change the return there to true or remove the functions because it gets the entering behavior from OpenDoors

Reply:
What do you think?

Reply:
@niccokunzmann I thought about this. But the treasure is meant to be exclusive and to be accessed only from that little crack on the wall of the room which has the key, So I changed the wall to a door.

Reply:
ok. Would you, since you show the wall, also block movement through the wall in the tile which has the wall? This is useful if the tile is used somewhere else, too.

Reply:
@niccokunzmann It is already there. The movement was blocked from both sides already.

See this : https://github.com/duttaditya18/labyrinth/blob/c1ebe5af2fe896f41db7ae5fb977fd65c377f62a/js/tiles.js#L905

Reply:
I see this:

```
        canLeaveToTheRight: function(player){
            return true;
},
```
which should be 

```
        canLeaveToTheRight: function(player){
            return false;
},
        canEnterFromTheRight: function(player){
            return false;
},
```

to block the movement

Reply:
Done! Sorry for the delay!